vnext/Microsoft.ReactNative/IReactModuleBuilder.idl

```diff
@@ -7,55 +7,58 @@ import "IReactContext.idl";
+namespace Microsoft.ReactNative
+{
+  DOC_STRING(
+    "A delegate that will set `reactContext` for a module.  "
+    "We use it for a stand-alone initialize method, strongly typed JS events and functions."
+    "Experimental code uses it to initialize TurboModule `CallInvoker`.")
   delegate void InitializerDelegate(IReactContext reactContext);
 
   // Native method return type.
-  enum MethodReturnType {
+  enum MethodReturnType
+  {
     Void,
     Callback,
     TwoCallbacks,
     Promise,
   };
 
-  // A callback to call JS code with results.
   DOC_STRING("A callback to call JS code with results.")
   delegate void MethodResultCallback(IJSValueWriter outputWriter);
 
-  // A delegate to call native asynchronous method.
   DOC_STRING("A delegate to call native asynchronous method.")
   delegate void MethodDelegate(
       IJSValueReader inputReader,
       IJSValueWriter outputWriter,
       MethodResultCallback resolve,
       MethodResultCallback reject);
 
-  // A delegate to call native synchronous method.
   DOC_STRING("A delegate to call native synchronous method.")
   delegate void SyncMethodDelegate(IJSValueReader inputReader, IJSValueWriter outputWriter);
 
-  // A delegate to gather constants from native modules.
-  DOC_STRING("A delegate to gather constants from native modules.")
+  DOC_STRING("A delegate to gather constants for the native module.")
   delegate void ConstantProviderDelegate(IJSValueWriter constantWriter);
 
-  // Builds native modules inside of React Native code based on the provided meta-data.
   [webhosthidden]
-  DOC_STRING("Builds native modules inside of React native code based on the provided meta-data. \
-See [Native Modules](native-modules) for more usage information.")
-  interface IReactModuleBuilder {
-
-    DOC_STRING("An initializer is a method that will be called when the react instance starts.  It provides the native module with the @IReactContext for the running instance. See @InitializerDelegate.")
+  DOC_STRING(
+    "Builds native module inside of ReactNative code based on the provided meta-data.  "
+    "See [Native Modules](native-modules) for more usage information.")
+  interface IReactModuleBuilder
+  {
+    DOC_STRING(
+      "Add a initializer method called on the native module initialization.  "
+      "It provides the native module with the @IReactContext for the running ReactNative instance.  See @InitializerDelegate."
+      "There can be multiple initializer methods which are called in the order they were registered.")
     void AddInitializer(InitializerDelegate initializer);
 
-    DOC_STRING("The JavaScript values written by the @ConstantProviderDelegate will be available as constants on the native module is JavaScript.")
+    DOC_STRING("Add a constant provider method to define constants for the native module.  See @ConstantProviderDelegate.")
```

Comment:
NB: When describing an API, prefer phrasing the doc string using a description rather than an imperative/command. For example "Adds a constant provider" instead of "Add a constant provider".

(same in other APIs)

Reply:
Fixed.